Unit test fails for model User

One of the tests fails for model User:

On mean commit: 4c18d996a62d4066f61276f19ec23ad3e6cddf8f
Other versions:
grunt@0.4.2
mocha@1.17.1

Test failure:

<Unit Test>
    Model User:
      Method Save
        ✓ should begin with no users
        ✓ should be able to save whithout problems
        ✓ should fail to save an existing user again
        1) should be able to show an error when try to save without name

  5 passing (334ms)
  1 failing

  1) <Unit Test> Model User: Method Save should be able to show an error when try to save without name:
     Uncaught AssertionError: expected null to exist
      at Promise.<anonymous> (/Users/km/dev/mean/test/mocha/user/model.js:56:28)
      at Promise.<anonymous> (/Users/km/dev/mean/node_modules/mongoose/node_modules/mpromise/lib/promise.js:177:8)
      at Promise.EventEmitter.emit (events.js:98:17)
      at Promise.emit (/Users/km/dev/mean/node_modules/mongoose/node_modules/mpromise/lib/promise.js:84:38)
      at Promise.fulfill (/Users/km/dev/mean/node_modules/mongoose/node_modules/mpromise/lib/promise.js:97:20)
      at handleSave (/Users/km/dev/mean/node_modules/mongoose/lib/model.js:133:13)
      at /Users/km/dev/mean/node_modules/mongoose/lib/utils.js:408:16
      at /Users/km/dev/mean/node_modules/mongoose/node_modules/mongodb/lib/mongodb/collection/core.js:637:9
      at Server.Base._callHandler (/Users/km/dev/mean/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/base.js:442:41)
      at /Users/km/dev/mean/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/server.js:485:18
      at MongoReply.parseBody (/Users/km/dev/mean/node_modules/mongoose/node_modules/mongodb/lib/mongodb/responses/mongo_reply.js:68:5)
      at null.<anonymous> (/Users/km/dev/mean/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/server.js:443:20)
      at EventEmitter.emit (events.js:95:17)
      at null.<anonymous> (/Users/km/dev/mean/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/connection_pool.js:191:13)
      at EventEmitter.emit (events.js:98:17)
      at Socket.<anonymous> (/Users/km/dev/mean/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/connection.js:418:22)
      at Socket.EventEmitter.emit (events.js:95:17)
      at Socket.<anonymous> (_stream_readable.js:746:14)
      at Socket.EventEmitter.emit (events.js:92:17)
      at emitReadable_ (_stream_readable.js:408:10)
      at emitReadable (_stream_readable.js:404:5)
      at readableAddChunk (_stream_readable.js:165:9)
      at Socket.Readable.push (_stream_readable.js:127:10)
      at TCP.onread (net.js:526:21)

--- a/app/models/user.js
+++ b/app/models/user.js
@@ -11,7 +11,10 @@ var mongoose = require('mongoose'),
  * User Schema
  */
 var UserSchema = new Schema({
-    name: String,
+    name: {
+        type: String,
+        required: true
+    },
     email: String,
     username: {
         type: String,

